### DISCUSSION TITLE
Setup.py file is missing in autosar package v0.5.3

### DISCUSSION THREAD

**Discussion by kaoutarelghaich** [cogu/autosar#119] [General]
Hello,

Could you please add the setup.py file to the latest version of autosar package (v0.5.3) ? This file is mandatory for the manual install option in python.

Thank you,
**EL Ghaich Kaoutar**

**Comment by cogu** [comment]
In modern Python the setup.py file is no longer required. The new way to manually install projects is something like:

`python -m pip install .`

For more information see the [Official Python Packaging User Guide](https://packaging.python.org/en/latest/guides/modernize-setup-py-project/).

**Reply by kaoutarelghaich** [reply]
Unfortunatelly, the pip install is not supported yet by python as shown down below. This is found in the documentation https://autosar.readthedocs.io/en/latest/start.html

**Reply by cogu** [reply]
What do you mean by "not supported yet by Python"?

The documentation you're referencing is for Python AUTOSAR v0.4 which is no longer in development and incompatible with v0.5.3. 

Python AUTOSAR v0.5.3 is a work in progress and doesn't yet have any documentation.

This is all clearly described in the project README.
